nrgpy/utils/utilities.py

```diff
@@ -111,17 +104,42 @@ def date_check(start_date, end_date, string):
         start = start_date
         end = end_date
 
-    try:
-        date_text = re.search(date_format, string)
-        file_date = datetime.strptime(date_text[0], strp_format)
+    if re.search(date_format_no_dash, string):
+        date_text = re.search(date_format_no_dash, string)
+        try:
+            file_date = datetime.strptime(date_text[0], strp_format_no_dash)
+            if (file_date >= start) and (file_date <= end):
```

Comment:
can we put this 
```
if (file_date >= start) and (file_date <= end):
    return True
else:
    return False
```
outside of this if/else and try/except clauses?

Reply:
yes, see updated code (tests still pass)

Reply:
ok i meant outside of the whole if/else clause

Reply:
right. i knew that. yes. done.